uiServices/test/configFile.feature

```diff
@@ -360,7 +356,7 @@ Feature: Create Configuration File
     Given the config file "GodogConfig.xml" exists at "/uiServices/test/"
     Given the automated performance ui server is available
     And the header configsDirPath is "/uiServices/test/"
-    When I send "PUT" request to "/configs/GodogConfig" with body:
+    When I send "PUT" request to "/configs/GodogConfig.xml" with body:
```

Comment:
not sure i'm happy .xml is being passed down. should we just pass down the name and assume xml? perhaps the server could assume xml, and we could pass down a header if this ever changed in the future.

Reply:
see email about this one!